core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -141,6 +148,21 @@ public static LocalCacheManager create(CacheManagerOptions options,
     mInitService =
         options.isAsyncRestoreEnabled() ? Optional.of(Executors.newSingleThreadExecutor()) :
             Optional.empty();
+    if (options.isTtlEnabled()) {
+      mTtlEnforcerExecutor = Optional.of(newScheduledThreadPool(1));
+      mTtlEnforcerExecutor.get().scheduleAtFixedRate(() ->
+              LocalCacheManager.this.invalidate(pageInfo -> {
+        try {
+          return System.currentTimeMillis() - pageInfo.getCreatedTimestamp() >=
+                  options.getTtlThresholdSeconds() * 1000;
+        } catch (Exception ex) {
+          // In case of any exception, do not invalidate the cache
+          return false;
+        }
+      }), 0, options.getTtlCheckIntervalSeconds(), SECONDS);
+    } else {
+      mTtlEnforcerExecutor = Optional.empty();
+    }
```

Comment:
The indentation here looks a bit weird. Would you mind double-checking whether this fulfills the coding style by running "mvn checkstyle:checkstyle"?